### DISCUSSION TITLE
Terminal run error(FedCE: Fair Federated Learning via Client Contribution Estimation)

### DISCUSSION THREAD

**Discussion by HTQ3537** [NVIDIA/NVFlare#2000] [Q&A]
In the terminal type:
nvflare simulator jobs/fedce_prostate -w /tmp/nvflare/fedce_prostate -c client_I2CVB,client_MSD,client_NCI_ISBI_3T,client_NCI_ISBI_Dx,client_Promise12,client_PROSTATEx -gpu 0,1,0,1,0,1
Terminal run error:ModuleNotFoundError: No module named 'learners.supervised_monai_prostate_fedce_learner'
![473d10f81feb5494d3d78c782d22e7e](https://github.com/NVIDIA/NVFlare/assets/62830446/55606aa3-209c-4aa1-a9bb-19ca315b7d9e)

**Comment by YuanTingHsieh** [comment]
@HTQ3537 thanks for reporting.

Can you provide your OS, python version, NVFlare version and which NVFlare branch that your are using?

@ZiyueXu77 have you seen this before?

**Reply by ZiyueXu77** [reply]
Seems a path issue, @HTQ3537 could you check if the custom folder has been sent to /app_client_Promise12?

**Reply by HTQ3537** [reply]
@YuanTingHsieh I'm using ubantu, python 3.10, NVFlare 2.3.0.What is NVFlare branch, which I started by typing a command in my terminal.
@ZiyueXu77 I can't find the /app client Promise12 folder

**Reply by YuanTingHsieh** [reply]
@HTQ3537 
After you clone the NVFlare GitHub repo.
Please do checkout to 2.3 branch: (`git checkout 2.3`)
And please install latest 2.3 version using: (`python3 -m pip install nvflare==2.3.8`)

Regarding that folder, can you type the following command on your machine:
`ls /tmp/nvflare/fedce_prostate/simulate_job/app_client_Promise12/custom`

Can you also type the following command and paste the results:
`python3 -m pip list`

**Reply by YuanTingHsieh** [reply]
@HTQ3537 from other users feedback: https://github.com/NVIDIA/NVFlare/discussions/2155

I will suggest you try NVFlare 2.4.0rc7 (`pip install nvflare~=2.4.0rc`)
And use example in the NVFlare 2.4 branch (`git checkout 2.4`)
